src/cli/index.js

```diff
@@ -398,14 +399,32 @@ export function main({
     });
 }
 
-export default function start() {
-  // ignore all arguments after a --
-  const doubleDashIndex = process.argv.findIndex(element => element === '--');
-  const startArgs = process.argv.slice(0, 2);
-  const args = process.argv.slice(2, doubleDashIndex === -1 ? process.argv.length : doubleDashIndex);
-  const endArgs = doubleDashIndex === -1 ? [] : process.argv.slice(doubleDashIndex + 1, process.argv.length);
+async function start(): Promise<void> {
```

Comment:
I think it's fine as is, but if you prefer it another way I can change it (note that the symbol is still exported [here](https://github.com/yarnpkg/yarn/pull/4204/files/890bab1bd8e0767704bf29bb1b908d500369a10c#diff-867becf4a9c2c6c6d4e7c1278750724eR438), it's just that the function declaration and symbol export have to be split for Flow to correctly parse it).